vignettes/cn_sv_tutorial.Rmd

```diff
@@ -7,11 +7,10 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 ```{r setup, include = FALSE}
-knitr::opts_chunk$set(warning = FALSE, message = FALSE)
```

Comment:
out of curiousity and for the loooooong time in the future - can we replace the Rmds with qmd files? Then the documents are so much cleaner 😀